packages/components/pie-chip/src/index.ts

```diff
@@ -11,10 +15,82 @@ const componentSelector = 'pie-chip';
 
 /**
  * @tagname pie-chip
+ * @slot icon - The icon slot
+ * @slot - Default slot
  */
 export class PieChip extends LitElement implements ChipProps {
+    @property()
+    @validPropertyValues(componentSelector, variants, 'primary')
+    public variant: ChipProps['variant'] = 'default';
+
+    @property({ type: Boolean })
+    public disabled = false;
+
+    @property({ type: Boolean })
+    public isSelected = false;
```

Comment:
Do these all need to be marked as optional `?` to enforce the interface types? If not we can probably clean up all our other components 😂

Reply:
imo, I don't think we need to mark it as optional since we've assigned a default value and the prop will always have/need a value within the class implementation. the prop is indeed optional for consumers, but not for the class itself. That's different for things where the class expects it to be optional/ undefined such as `href`

I think the key here is to realize that what's optional for consumers doesn't necessarily correspond to the class contract, that's why the other day I suggested the idea of having a contract/interface for consumers and one for the class implementation but not sure how to procced with that xD